OptimoveSDK/optimove-sdk/src/main/java/com/optimove/sdk/optimove_sdk/kumulos/PushSubscriptionManager.java

```diff
@@ -0,0 +1,488 @@
+package com.optimove.sdk.optimove_sdk.kumulos;
+
+import android.content.Context;
+import android.net.Uri;
+import android.text.TextUtils;
+
+import androidx.annotation.Nullable;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import okhttp3.Call;
+import okhttp3.Callback;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+
+/**
+ * Allows managing the push subscriptions for this installation ID & creating channels from the SDK
+ */
+public class PushSubscriptionManager {
```

Comment:
from what i've heard beacons, geofences and channel management didnt make it to the Optimove SDK :(

Hence, some unneeded classes: `PushSubscriptionManager`, `PushChannel`, publicAPI: `sendLocationUpdate`, `trackEddystoneBeaconProximity` shall be removed

Reply:
correct, over time they may come back - but its better to re-add once they work, rather than have clients try to use them while they dont